cli/src/commands/init.rs

```diff
@@ -78,14 +78,20 @@ pub struct Opts {
 
 impl Opts {
 	pub async fn execute(&self, term: &Term) -> GlobalResult<()> {
-		// Remove legacy `.rivet` dir if exists
+		// Remove legacy `.rivet/config.[yaml|toml|json]` files if they exists
 		let legacy_project_meta_path = paths::project_root()?.join(".rivet");
-		if fs::metadata(&legacy_project_meta_path).await.is_ok() {
+		if let Ok(mut entries) = fs::read_dir(&legacy_project_meta_path).await {
 			term::status::warn(
 				"Deleting legacy project metadata",
-				".rivet/ folder is moved to a global config",
+				".rivet/config.[yaml|toml|json] file is moved to a global config",
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XiP77c3NM4Q27xw9sqIb/28cebbd5-00bf-45c5-a478-4dfd80ebcb3b.png)

These are the files that got added from Godot (ignore `config.yaml`). I want to double-check that there is no way for them to accidentally ship `config.gd`.